### DISCUSSION TITLE
Console printout

### DISCUSSION THREAD

**Discussion by styyokuda** [idaholab/moose#25632] [Q&A General] [answered]
All,

I am using the method shown below for the console printout.  In this manner, the log information for all processors used for a run is printed in the console output file; namely, the same printout is repeated as many as processors used.  Is there any way the log information for only one processor is printed without repeating the log information for the rest of the processors?
Please help.
S. Thomas
```
[Outputs]
       . 
       .
  [./console]
    type = Console
    output_file = true
  [../]
       .
       .
[ ]
```

**Comment by GiudGiud** [comment]
Hello

It should already be that way.
What does the header say at the top of the simulation for the number of processes?

Guillaume

**Reply by styyokuda** [reply]
Thank you, Guillaume.
It reads
Parallelism:
  Num Processors: 6
  Num Threads:      1

Does this help?
S. Thomas

**Reply by GiudGiud** [reply]
Ok this is normal.

You should not get output from each process in this configuration. Unless you pass --keep-coût on the command line you should only see output from rank 0

**Reply by styyokuda** [reply]
Thank you, Guillaume.

Yes that is true for the printout on the "console."  My question is the printout in the console output "file" as "output_file = true."  Is it supposed that the same printout is repeated as many as processors used in the output file?

I apologize that I confused you.
Please help.
S. Thomas

**Reply by GiudGiud** [reply]
ah ok I guess that doesnt work?

do this instead:
`--keep-cout --redirect-stdout` and that will redirect the console output to a bunch of files

**Reply by styyokuda** [reply, marked as answer]
Thank you, Guillaume.
I think Linux commend ">" works.
S. Thomas
